packages/endpoints-overlay/src/EndpointsOverlay.story.tsx

```diff
@@ -95,3 +96,17 @@ export const EndpointsOverlayWithUnnamedPlace: ComponentStory<typeof EndpointsOv
     toLocation={unnamedToLocation}
   />
 );
+
+export const EndpointsOverlayWithStopLink: ComponentStory<typeof EndpointsOverlay> = () => (
```

Comment:
Remove this story. The View Nearby link already appears in story "Endpoint Overlay With User Settings" and "Endpoint Overlay with Unnamed Place". Pass the new `setViewedStop` action declared on line 17 to those existing stories.

Reply:
Whoops thank you for this catch!